platform/install/environments/gcp.mdx

```diff
@@ -0,0 +1,292 @@
+
+## Prerequisites
+
+Before starting, ensure you have the following tools installed:
+<BasePrerequisites />
+- vCluster CLI <InstallCli />
+- [Google Cloud SDK (`gcloud` CLI)](https://cloud.google.com/sdk/docs/install)
+  :::note
+  Ensure you have the necessary IAM permissions to create clusters and manage cloud services.
+  :::
+
+## Create GKE cluster
+
+<Flow id="create-gke-cluster">
+
+<Step>
+Prepare the environment.
+
+Start by creating a zonal GKE cluster using the `gcloud` CLI. First, set up your environment variables:
+
+:::tip
+Project ID can be found in the Google Cloud Console under the project name.
+Alternatively use `gcloud project list` to list all projects and their IDs.
+To check which project is active, use `gcloud config get-value project`.
+:::
+
+```bash title="Set environment variables"
+export PROJECT_ID=development
+export CLUSTER_NAME=vcluster-demo
+export ZONE=europe-west1-b
+export MACHINE_TYPE=e2-standard-4
+```
+
+Configure `gcloud` and enable the required APIs and set default project:
+
+```bash title="Configure gcloud"
+gcloud config set project $PROJECT_ID
+gcloud services enable container.googleapis.com
+```
+</Step>
+
+<Step>
+Create the cluster.
+
+```bash title="Create GKE cluster"
+gcloud container clusters create $CLUSTER_NAME \
+    --zone $ZONE \
+    --machine-type $MACHINE_TYPE \
+    --num-nodes 2
+```
+
+  :::info
+  This process typically takes about 10-15 minutes.
+  :::
+
+This command creates a GKE cluster named vcluster-demo in the europe-west1-b
+zone with two nodes of type e2-standard-4.
+
+<KubeconfigUpdate />
+
+</Step>
+
+<Step>
+Verify the cluster creation.
+
+Verify that the GKE was created successfully by listing the nodes:
+
+```bash title="List cluster nodes"
+kubectl get nodes
+```
+
+You should see output similar to:
+```
+NAME           LOCATION        MASTER_VERSION      MASTER_IP      MACHINE_TYPE   NODE_VERSION        NUM_NODES  STATUS
+vcluster-demo  europe-west1-b  1.30.5-gke.1443001  35.187.66.218  e2-standard-4  1.30.5-gke.1443001  2          RUNNING
+```
+<ManagedK8sInstallVcluster />
+
+</Step>
+
+</Flow>
+
+## Platform setup
+
+With the GKE cluster up and running, you can now deploy the platform in a few
+easy steps.
+
+<Flow>
+
+### Install the platform
+
+<Step>
+Deploy platform using the vCluster CLI
+
+Now that you have vCluster running on GKE, set up the [platform
+](/platform/install/quick-start-guide) and configure GPC in the following steps.
+
+Easiest way to deploy a platform is using the `vcluster CLI`
+
+```bash title="deploy the platform using vcluster cli
+vcluster platform start
+```
+The command asks you for providing email address for the admin user
+
+```bash title="deployment expected output"
+By providing your email, you accept our Terms of Service and Privacy Statement:
+Terms of Service: https://www.loft.sh/legal/terms
+Privacy Statement: https://www.loft.sh/legal/privacy
+? Please specify an email address for the admin user
+```
+
+:::tip retry
```

Comment:
Can we mention that the operation is idemopotent? That it, if the cluster already exists, creating it again will do nothing? In retry operations, it's very important, as it avoids adding a guard condition `if cluster does not exist` and running into race conditions.